### PR TITLE
Fix duplicate certificate list item (EXPOSUREAPP-11052)

### PR DESCRIPTION
**Pull request by marcauberer**
Addresses [EXPOSUREAPP-11052](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-11052).

This fixes the edge case that a RA test certificate and a PCR test certificate is required, but the certificate type list contains `PCR_TEST`, `RA_TEST` and `TEST`.

The previous result was:
```
PCR-Testzertifikat, Schnelltest-Testzertifikat,
PCR-Testzertifikat, Schnelltest-Testzertifikat
```

The new result should be:
```
PCR-Testzertifikat, Schnelltest-Testzertifikat
```
because `PCR_TEST` and `RA_TEST` are filtered as soon as `TEST` is present.

### CODE REVIEW

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/dccticketing/ui/certificateselection/DccTicketingCertificateSelectionHelperFunctions.kt [hunk 1]
```diff
@@ -10,19 +10,21 @@ import de.rki.coronawarnapp.dccticketing.core.certificateselection.DccTicketingC
 import de.rki.coronawarnapp.dccticketing.core.certificateselection.DccTicketingCertificatesFilterType.VACCINATION
 
 fun Context.certificateTypesText(certificateTypes: List<String>, separator: String = ", "): String =
```
Comment (LukasLechnerDev) [inline Corona-Warn-App/src/main/java/de/rki/coronawarnapp/dccticketing/ui/certificateselection/DccTicketingCertificateSelectionHelperFunctions.kt:12]: Hi Marc! I am Lukas, and I started on Dec 1st in the CWA Android Team 👋.

A unit test for this extension function that fails before this fix and is successful after this fix would be very useful here I think.

**Review by mtwalli — DISMISSED** [review]
Thanks @marcauberer!

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/dccticketing/ui/certificateselection/DccTicketingCertificateSelectionHelperFunctions.kt (hunk 1 above)
Comment (mtwalli) [reply Corona-Warn-App/src/main/java/de/rki/coronawarnapp/dccticketing/ui/certificateselection/DccTicketingCertificateSelectionHelperFunctions.kt:12]: Added

**Review by mtwalli — APPROVED** [review]

**Review by LukasLechnerDev — APPROVED** [review]
LGTM!

**Comment by marcauberer** [comment]
@mtwalli thanks for the tests 👍

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/dccticketing/ui/certificateselection/DccTicketingCertificateSelectionHelperFunctions.kt (hunk 1 above)
Comment (marcauberer) [reply Corona-Warn-App/src/main/java/de/rki/coronawarnapp/dccticketing/ui/certificateselection/DccTicketingCertificateSelectionHelperFunctions.kt:12]: Hi @LukasLechnerDev! I am Marc, helping the CWA Android Team here and there with fixing bugs ...
